package.json

```diff
@@ -103,5 +103,8 @@
     "extends": [
       "plugin:storybook/recommended"
     ]
+  },
+  "dependencies": {
+    "react-responsive-carousel": "^3.2.23"
```

Comment:
react-responsive-carousel must be a dependency for the package and not for the entire monorepo